client/src/components/Map.js

```diff
@@ -110,12 +113,25 @@ class Map extends Component {
 
     if (hexX === this.state.hexX && hexY === this.state.hexY) return
```

Comment:
What's the purpose of this short-circuit? Is it needed because of React's diffing algorithm?

Reply:
This is so the map doesn't redraw when the cursor is moving within a hex.